src/Components/ReusableComponents/Header/Header.css

```diff
@@ -0,0 +1,39 @@
+.header {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
```

Comment:
This is tiny, but I _think_ that assigning fixed fractions here might yield more consistent results - maybe 1fr 3fr 1fr. That way I think those boxes should take up pretty much the same space across screens.